app/src/main/java/org/simple/clinic/drugstockreminders/DrugStockReminderApi.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.drugstockreminders
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface DrugStockReminderApi {
+
+  @GET("v4/drug_stock")
+  fun drugStockReminder(
+      @Query("date") previousMonth: String
```

Comment:
We cannot pass the `LocalDate` as query param directly?

Reply:
We are passing the entire date, so let's keep the name of the param as `date`

Reply:
We can, but the requested param is a string.

Reply:
I know, just checking if we have verified whether Retrofit will automatically convert the `LocalDate` to string. Since the date string, we need to pass is basically `LocalDate#toString`. It's not that big of a deal, we can merge this in. I just wanted to check if there was an issue with `LocalDate`.

Let's update the param name as the API definition just expects a date, it doesn't care if it's last month or current month. So, let's rename this to `date`, WDYT?

Reply:
Right so I was thinking for the name to be something that we're sending, more readable. 
I'll check the issue with the Localdate/string in the implementation PR.

Reply:
I am merging this in for now, we can make this change later.

Reply:
So the whole point of adding @json strings explicitly is because we want to rename it in the variable and fit in our use-case. Hence this makes more sense to me